cg/cli/upload.py

```diff
@@ -314,6 +315,8 @@ def observations(context, case_id, dry_run):
             LOG.info("%s: observations uploaded!", family_obj.internal_id)
         except DuplicateRecordError as error:
             LOG.info("skipping observations upload: %s", error.message)
+        except CalledProcessError:
```

Comment:
Please consider if can make the duplicationerror handling explicit and handle unexpected exceptions separately

Reply:
@patrikgrenfeldt, I've added a feature in loqusdb that explicitly asks if a duplicate exists in the database. This is used in the loqusdb API in cg, and if a duplicate is found, an exception is raised and will be handled.